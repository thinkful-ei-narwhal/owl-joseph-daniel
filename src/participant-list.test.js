import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ParticipantList from './participant-list';
import STORE from './store';



describe('<ParticipantList />', () => {
  it('smoke test', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ParticipantList participants={STORE.participants}/>, div);
    // if something is wrong, itd fail here
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders this UI as expected', () => {
    const tree = renderer.create(<ParticipantList participants={STORE.participants}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});