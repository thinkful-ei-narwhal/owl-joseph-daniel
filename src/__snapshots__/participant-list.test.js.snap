// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ParticipantList /> renders this UI as expected 1`] = `
<div
  className="participant-list"
>
  <div
    className="Participant"
  >
    <img
      alt="Koren Templeton"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <h3>
      Koren Templeton
    </h3>
    <p>
      in session
    </p>
  </div>
  <div
    className="Participant"
  >
    <img
      alt="Caty Flucker"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
    <h3>
      Caty Flucker
    </h3>
    <p>
      in session
    </p>
  </div>
  
  <div
    className="Participant"
  >
    <img
      alt="Frank Runciman"
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
    <h3>
      Frank Runciman
    </h3>
    <p>
      in session
    </p>
  </div>
  <div
    className="Participant"
  >
    <img
      alt="Ashla Attwool"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <h3>
      Ashla Attwool
    </h3>
    <p>
      in session
    </p>
  </div>
</div>
`;
